cocos/renderer/backend/opengl/TextureGL.cpp

```diff
@@ -80,13 +92,13 @@ TextureGL::TextureGL(const TextureDescriptor& descriptor) : Texture(descriptor)
 {
     glGenTextures(1, &_texture);
     toGLTypes();
-    
+    bool bPow2 = ISPOW2(_width) && ISPOW2(_height);
```

Comment:
We don't use Hungary style, so it is better to change to `isPow2`.

Reply:
got it.